plugins/stylelint-no-missing-parenthesis/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "stylelint-no-missing-parenthesis",
+  "version": "1.0.1",
+  "license": "MIT",
+  "author": "Rajdeep Chandra",
+  "main": "index.js",
+  "scripts": {
+    "lint": "stylelint index.js",
+    "prepublishOnly": "yarn test",
+    "test": "ava"
+  },
+  "peerDependencies": {
+    "stylelint": ">=15"
+  },
+  "devDependencies": {
+    "ava": "^3.9.0",
+    "stylelint": ">=15"
```

```suggestion
    "stylelint": "^15.0.0"
```

I'm not sure but I don't think >= works in devDeps

---

Umm I think it works but I made the changes to its latest versions.